pkg/kapp/clusterapply/delete_change.go

```diff
@@ -32,10 +33,40 @@ type DeleteChange struct {
 	identifiedResources ctlres.IdentifiedResources
 }
 
+type uniqueResourceRef struct {
+	schema.GroupKind
+	Name string
+}
+
+// if any new reource will encountered which kapp can not delete and required to orphaned then resource info will be added here.
```

Comment:
nit: (just trying to make it a bit clearer)
```suggestion
// List of resources use the "orphan" delete strategy implicitly as they cannot be deleted by end users
```

Reply:
Updated